distributed/utils.py

```diff
@@ -1296,7 +1297,9 @@ class Log(str):
     """ A container for logs """
 
     def _repr_html_(self):
-        return "<pre><code>\n{log}\n</code></pre>".format(log=self.rstrip())
+        return "<pre><code>\n{log}\n</code></pre>".format(
+            log=html.escape(self.rstrip())
```

Comment:
The change discovered a missing HTML escaping during the UI rendering. This fixes the test for me locally. Let's see if Travis agrees